schema/musicxml.xsd

```diff
@@ -2731,6 +2731,12 @@ These letter dynamic symbols are separated from crescendo, decrescendo, and wedg
 				</xs:annotation>
 			</xs:element>
 
+			<xs:element name="for-part" type="for-part" minOccurs="0">

```

Comment:
Could be an optional member of the `<transpose>` type instead or a member of `<attributes>`?

Reply:
@webern Thanks for the suggestion. Since we are including both transpose and clef information, it seemed better to put this at the `<attributes>` level.  This also seems to be more extensible if we want to add other attributes in the future. Are there scenarios where you believe that adding this at the `<transpose>` element level would make things work better or easier?

Reply:
I thought it might be a little bit easier for parsing, but I could be wrong. In general I'm not crazy about the MusicDataChoice looping and reacting, but it's how we already do things so it's just another `case:`... It works!